packages/tocco-ui-showcase/src/toccoUtilExamples/notifier/example.js

```diff
@@ -97,16 +105,41 @@ class Example extends React.Component {
         <div>
           <notifier.Notifier/>
           <div>
-            <button className="btn btn-info" onClick={this.info}>Info</button>
-            <button className="btn btn-success" onClick={this.success}>Success (no timeout)</button>
-            <button className="btn btn-warning" onClick={this.warning}>Warning (long)</button>
-            <button className="btn btn-danger" onClick={this.error}>Error</button>
-            <button className="btn btn-info" onClick={this.blockingInfo}>Blocking Info</button>
+            <Button
+              label="Info"
+              look="raised"
+              onClick={this.info}/>&nbsp;
```

Comment:
Personally I would abstain &nbsp; and {style: ...}. Since it was requested to be able to distinguish easier between each button, there are several implementation options. The closest variant to the framework would be to utilize <ButtonGroup>. Since this is an example, I prefer to be as concise as possible, without unneeded footprint. IMHO this is "&nbsp;".